flink/v1.19/flink/src/test/java/org/apache/iceberg/flink/SimpleDataUtil.java

```diff
@@ -82,6 +83,13 @@ private SimpleDataUtil() {}
           Types.NestedField.optional(1, "id", Types.IntegerType.get()),
           Types.NestedField.optional(2, "data", Types.StringType.get()));
 
+  public static final Schema SCHEMA_PRI =
```

Comment:
could we use a more descriptive name, like `SCHEMA_WITH_PRIMARY_KEY`?

Reply:
Fixed.